### PR TITLE
[PPML] Fix Scala version related issue realtime serving

### PR DESCRIPTION
**Pull request by Uxito-Ada**
1. Convert SPART_VERSION from 3.1.2 to 2.4.6, which can satisfy the dependency requirement.
2. Change the success condition in the check file, according to the changed output message of latest flink.
3. Change the jar file paths, thus it will work in the current image.
4. Append redis host value which has been missed.

### CODE REVIEW

File: ppml/trusted-realtime-ml/scala/docker-graphene/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ARG BIGDL_VERSION=0.14.0-SNAPSHOT
-ARG SPARK_VERSION=3.1.2
+ARG SPARK_VERSION=2.4.6
```
Comment (glorysdj) [inline ppml/trusted-realtime-ml/scala/docker-graphene/Dockerfile:2]: keep 3.1.2 and update all the scala version to 2.12

File: ppml/trusted-realtime-ml/scala/docker-graphene/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 
 # stage.1 graphene
 FROM ubuntu:18.04 AS graphene
```
Comment (qiyuangong) [inline ppml/trusted-realtime-ml/scala/docker-graphene/Dockerfile:5]: Need to add SCALA_VERSION env to control flink download link

**Review by qiyuangong — APPROVED** [review]
LGTM

**Review by qiyuangong — APPROVED** [review]
LGTM

**Comment by qiyuangong** [comment]
Merge blocked by conflict.
@Uxito-Ada Need to rebase with branch 2.0

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**